src/main/kotlin/me/gserv/fabrikommander/commands/TpaHereCommand.kt

```diff
@@ -0,0 +1,49 @@
+package me.gserv.fabrikommander.commands
+
+import me.gserv.fabrikommander.data.TeleportRequest
+import me.gserv.fabrikommander.utils.Context
+import me.gserv.fabrikommander.utils.Dispatcher
+import me.gserv.fabrikommander.utils.aqua
+import me.gserv.fabrikommander.utils.plus
+import me.gserv.fabrikommander.utils.red
+import me.gserv.fabrikommander.utils.hover
+import me.gserv.fabrikommander.utils.click
+import me.gserv.fabrikommander.utils.reset
+import net.minecraft.command.argument.EntityArgumentType
+import net.minecraft.server.command.CommandManager.argument
```

Comment:
Forgot ClickEvent and HoverEvent imports o_o
```suggestion
import net.minecraft.server.command.CommandManager.argument
import net.minecraft.text.ClickEvent
import net.minecraft.text.HoverEvent
```